docs/FeatureManagement/FeatureFlag.v1.0.0.schema.json

```diff
@@ -76,8 +76,8 @@
                 "description": "The name used to refer to and require a client filter.",
                 "default": "",
                 "examples": [
-                  "Percentage",

```

Comment:
These are example names, not names of actual filters. I don't think we should propose in the schema the "Microsoft" name on filters. We wouldn't expect customers to use our prefix.